helm/ingress-nginx/templates/admission-webhooks/job-patch/psp.yaml

```diff
@@ -1,41 +1,52 @@
 {{- if (semverCompare "<1.25.0-0" .Capabilities.KubeVersion.Version) }}
-{{- if and .Values.controller.admissionWebhooks.enabled .Values.controller.admissionWebhooks.patch.enabled .Values.podSecurityPolicy.enabled (empty .Values.controller.admissionWebhooks.existingPsp) -}}
+{{- if and .Values.podSecurityPolicy.enabled .Values.controller.admissionWebhooks.enabled .Values.controller.admissionWebhooks.patch.enabled (empty .Values.controller.admissionWebhooks.existingPsp) -}}
```

Comment:
Remember that for our specific case, we need `Values.podSecurityPolicy.enabled` to be `not Values.global.podSecurityStandards.enforced`

Reply:
I'm aware of that, this will be added later in a separate PR since I probably can not propose this to upstream (would not get accepted for being too custom/specific).

Reply:
By the way: Are we really using `global`? I remember this being an internal / virtual path and only used for referencing values from the top-level values in sub-charts.

Reply:
Nevermind, read the docs: https://helm.sh/docs/chart_template_guide/subcharts_and_globals.